hwdb.d/60-autosuspend.hwdb

```diff
@@ -46,6 +46,16 @@
 usb:v058Fp9540*
  ID_AUTOSUSPEND=1
 
+#########################################
+# Framework
+#########################################
+
+# HDMI Expansion Card
+usb:v32acp0002*
+# DisplayPort Expansion Card
+usb:v32acp0003*
```

Comment:
are you sure these IDs are right?

```
Pattern 'v32acp0002*' is invalid: Expected W:(0123...), found 'a'  (at char 3), (line:1, col:4)
Pattern 'v32acp0003*' is invalid: Expected W:(0123...), found 'a'  (at char 3), (line:1, col:4)
```

Reply:
Perhaps not, below is what is from udevadm info for the HDMI expansion card. As the path can change based on which port the card is plugged into, I referred to the systemd hwdb docs.

The id seems to be made from the vendor id and product id, where:
32ac:0002
becomes
v32acp0002

If this is incorrect, please advise the correct formation of the id?

Much appreciated!

P: /devices/pci0000:00/0000:00:08.3/0000:c3:00.3/usb5/5-1
N: bus/usb/005/005
L: 0
E: DEVPATH=/devices/pci0000:00/0000:00:08.3/0000:c3:00.3/usb5/5-1
E: DEVNAME=/dev/bus/usb/005/005
E: DEVTYPE=usb_device
E: DRIVER=usb
E: PRODUCT=32ac/2/0
E: TYPE=0/0/0
E: BUSNUM=005
E: DEVNUM=005
E: MAJOR=189
E: MINOR=516
E: SUBSYSTEM=usb
E: USEC_INITIALIZED=14500648999
E: ID_VENDOR=Framework
E: ID_VENDOR_ENC=Framework
E: ID_VENDOR_ID=32ac
E: ID_MODEL=HDMI_Expansion_Card
E: ID_MODEL_ENC=HDMI\x20Expansion\x20Card
E: ID_MODEL_ID=0002
E: ID_REVISION=0000
E: ID_BUS=usb
E: ID_USB_INTERFACES=:110000:030000:
E: ID_PATH=pci-0000:c3:00.3-usb-0:1
E: ID_PATH_TAG=pci-0000_c3_00_3-usb-0_1
E: ID_FOR_SEAT=usb-pci-0000_c3_00_3-usb-0_1
E: TAGS=:seat:
E: CURRENT_TAGS=:seat: